### ISSUE TITLE
 implement classification head

### ISSUE THREAD

**Issue by RuiFilipeCampos** [Digital-Defiance/llm-voice-chat#25]
just a feed forward to project to the one hot encoding of the 5 stars instead of the token logits
